pkg/machinery/config/container/container.go

```diff
@@ -226,16 +226,22 @@ func (container *Container) Validate(mode validation.RuntimeMode, opt ...validat
 		warnings, err = container.v1alpha1Config.Validate(mode, opt...)
 	}
 
+	var multiErr *multierror.Error
```

Comment:
this is a bug I found with multi-doc config validation (unrelated to this PR), regression tests added